Testing/Exec/IgnitionDelay/exec_ignDelay.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
This looks like it's the same as the other sh script. Should we somehow consolidate?

Reply:
So the difference was that the `exec_ignDelay.sh` is the file that a user would use to compute ignition delay. The `githubTest.sh` was meant to do it AND check that the value obtained was reasonable. This was done by passing an argument to the `computeIgnitionDelay.py`.  I also think this was a bit redundant, and that the test should actually not be buried in the ignition delay computation.

I got rid of the two bash files and only use `exec_ignDelay.sh`. Now `computeIgnitionDelay.py` is not in charge of checking that the value is ok anymore. It output the value to a log file which is read and checked during the github action.